src/promo/DonationBanner.jsx

```diff
@@ -5,15 +5,16 @@ import Paper from '@material-ui/core/Paper'
 import PropTypes from 'prop-types'
 import React from 'react'
 import Typography from '../Typography'
+import corePalette from '../styles/palettes/core'
 import withStyles from '@material-ui/core/styles/withStyles'
 
 const styles = theme => ({
   // TODO: this is temporary while we work out what to do about
   // themes and palettes properly
   paper: {
     position: 'relative',
-    backgroundColor: theme.palette.core && theme.palette.core.main,
-    color: theme.palette.primary.contrastText,
+    backgroundColor: corePalette[600],
+    color: '#fff',
```

Comment:
In hindsight we might be better off using the more basic `Box` instead of `Paper` as the main wrapper of this component.

I think it would let us do away with most of these style overrides:
```jsx
<Box bgcolor="primary.main" color="primary.contrastText" p={4}>
  Some cool content
</Box>
```
That way we're not fighting the border radius/elevation either.

Reply:
Good call; switching to `Box` over `Paper` means we don't need to override `elevation` nor `borderRadius`